eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -150,6 +156,8 @@ class VolumePreparer:
             if action.type != \
                EosUpdaterUtil.FlatpakRemoteRefActionType.UNINSTALL:
                 autoinstalls.add(action.ref.ref.format_ref())
+            else:
+                autoinstalls.discard(action.ref.ref.format_ref())
```

Comment:
I have never encountered `set.discard` before! I have always used:

```python3
                autoinstalls -= {action.ref.ref.format_ref()}
```

when I've wanted this, which has always felt clunky. Nice.

Reply:
Yeah just discovered that too